src/scenes/Login/queries.graphql

```diff
@@ -0,0 +1,14 @@
+mutation Login($email: String!, $password: String!) {
+  login(input: { email: $email, password: $password }) {
```

Comment:
What do you think about using the input object instead, so fields don't have to be explicitly enumerated?
(That was the point of using input objects)
```suggestion
mutation Login($input: LoginInput!) {
  login(input: $input) {
```

Reply:
Good call!